config/jobs/kubernetes/sig-network/ingress-gce-e2e.yaml

```
@@ -175,7 +175,7 @@ postsubmits:
         - --env=REGISTRY=gcr.io/k8s-ingress-image-push
         - make
         - all-push
-        - ALL_ARCH='amd64 arm64'
+        - ALL_ARCH=amd64 arm64
```

I think that we need it with double quotes
```suggestion
        - ALL_ARCH="amd64 arm64"
```

```
```
W0306 13:30:51.697] /bin/sh: 1: Syntax error: Unterminated quoted string
W0306 13:30:51.697] make: *** [build/rules.mk:77: push-'amd64] Error 2
```